.circleci/config.yml

```diff
@@ -13,151 +13,121 @@ jobs:
         environment:
           MYSQL_ROOT_PASSWORD: mysqlpassword
 
-#     - image: microsoft/mssql-server-linux:ctp2-0
-#       environment:
-#         ACCEPT_EULA: 'Y'
-#         SA_PASSWORD: 'R@@tr@@t1234'
+      - image: microsoft/mssql-server-linux:2017-GDR
+        environment:
+          ACCEPT_EULA: 'Y'
+          SA_PASSWORD: 'Sqlboiler@1234'
 
     environment:
       GOPATH: /go
       ROOTPATH: /go/src/github.com/volatiletech/sqlboiler
 
     steps:
       - run:
-          name: Add PSQL Creds
+          name: 'Make GOPATH'
+          command: mkdir -p $ROOTPATH
+
+      - checkout:
+          name: 'Checkout'
+          path: /go/src/github.com/volatiletech/sqlboiler
+
+      # Workaround to allow the use of the circleci local cli.
```

Comment:
What's the point of this when the checkout step has already completed above? Just curious.

Reply:
It seems like circleci cli when running in a local environment checkouts (copies all the files) everything in `/root`.

```
[...]
====>> Checkout
  #!/bin/sh
mkdir -p /root && cp -r /tmp/_circleci_local_build_repo/. /root
[...]
```

So my strategy was to "in case checkout didn't work by checking if there isn't any file in there, use the local environment files"

But i think it would be a good idea to test it out directly in circleci once we merge this.